src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -15,10 +15,6 @@ PODVM_IMAGE ?= $(REGISTRY)/podvm-generic-$(PODVM_DISTRO)$(if $(filter $(SE_BOOT)
 PODVM_CONTAINER_NAME ?= $(REGISTRY)/podvm-docker-image-$(ARCH)
 VERIFY_PROVENANCE ?= no
 
-PUSH ?= false
```

Comment:
I suggest to leave this makefile toggle. We never know, in future we might decide to push the builder/binaries. Or developers can still use it to easily push and share with another dev.

Reply:
wouldn't a developer who is interested in this just push the image instead of using this toggle? I would generally lean towards removal of dead code, even if it could be useful [in the future](https://martinfowler.com/bliki/Yagni.html).


toggles like this incur a cost. coginitive burden for people who want to work on those build-scripts and often we end up propagating them in Dockerfile build-args/envs, workflow inputs, ... (cost of carry)

If we have a use case it should be trivial to re-add.

Reply:
Good points. I'm convinced this may incur in cost unnecessarily. Thanks @mkulke !